java/README.md

```diff
@@ -0,0 +1,308 @@
+This directory contains the java worker, with the following components.
```

Comment:
It'd be preferable to convert the documentation to RST (instead of markdown) for consistency with the rest of the documentation.

Reply:
It's ok to me, i could try to change the format from markdown to RST.